app/bundles/LeadBundle/Entity/CompanyRepository.php

```diff
@@ -126,12 +127,14 @@ public function getFieldGroups()
      * @param $leadId
      *
      * @return array
+     *
+     * @throws Exception
      */
     public function getCompaniesByLeadId($leadId, $companyId = null)
     {
         $q = $this->getEntityManager()->getConnection()->createQueryBuilder();
 
-        $q->select('comp.id, comp.companyname, comp.companycity, comp.companycountry, cl.is_primary')
+        $q->select('*')
```

Comment:
This doesn't seem optimal. I guess you need all columns from the companies table. Do you need also all columns from the companies_leads table?

Reply:
Yes, I needed fields from the companies table and an `is_primary` field from companies_leads.

Reply:
Would this work?
```suggestion
        $q->select('comp.*, cl.is_primary')
```